webapp/components/LoginForm/LoginForm.spec.js

```diff
@@ -0,0 +1,82 @@
+import { createLocalVue, mount } from '@vue/test-utils'
+import LoginForm from '@/components/LoginForm/LoginForm.vue'
+
+describe('Tests for NewsForm.vue', () => {
+  let wrapper
+
+  const localVue = createLocalVue()
+
+  const mutate = jest.fn().mockResolvedValue({ data: { login: '', signup: '' } })
+
+  const createComponent = (data) => {
+    wrapper = mount(LoginForm, {
+      localVue,
+      mocks: {
+        $apollo: {
+          mutate
+        },
+        $apolloHelpers: {
+          onLogin: jest.fn()
+        },
+        $store: {
+          commit: jest.fn()
+        },
+        $router: {
+          push: jest.fn
+        }
+      },
+      data () {
+        return {
+          ...data
+        }
+      }
+    })
+  }
+
+  afterEach(() => {
+    wrapper.destroy()
```

Comment:
I think if you return `wrapper` in your setup method above, you don't run into issues that you re-use a non-initialized `wrapper` by accident. Then you could do away with this `afterEach`.